docs/docusaurus/versioned_docs/version-1.4.0/orc8r/dev_security.md

```diff
@@ -34,13 +34,12 @@ Starting point for understanding the set of certificates related to Orc8r
         - When Orc8r self-signs its certificates, the `rootCA.key` will also be included in the deployment
     - `admin_operator.{key.pem,pem}` admin operator certificates for full access to northbound interface
 - Orc8r
-    - `controller.{key,crt}` Orc8r's server-validation certificate, signed by `rootCA.pem`
+    - `controller.{key,crt}` server-validation certificate for Orc8r and NMS, signed by `rootCA.pem`
     - `certifier.pem` Orc8r's client-validation certificate (root of trust)
     - `bootstrapper.key` Orc8r's signing key used in the bootstrap process
     - `fluentd.{key,pem}` Orc8r's certificates for its fluentd endpoints (fluentd is currently outside Orc8r proxy)
 - NMS
```

Comment:
Where do we have these certs ? We use the same admin operator certs which was used during seeding. We can probably avoid adding this one.